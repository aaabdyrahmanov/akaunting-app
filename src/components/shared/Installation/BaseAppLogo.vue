<script>
export default {
  name: 'BaseAppLogo',
  props: {
    logo: {
      type: Object,
      required: true
    }
  }
}
</script>

<template>
  <div class="header">
    <img
      :src="logo.src"
      :alt="logo.description"
      class="mb-3"
    >
  </div>
</template>

<style>
.header {
  -webkit-box-flex: 0;
  -ms-flex: 0 0 50%;
  flex: 0 0 50%;
  max-width: 50%;
  margin: auto;
  text-align: center;
}

.header img {
  width: 12%;
  margin-bottom: 3rem!important;
}
</style>