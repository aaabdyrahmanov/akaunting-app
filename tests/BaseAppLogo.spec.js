import { shallowMount } from '@vue/test-utils'
import BaseAppLogo from '../src/components/shared/Installation/BaseAppLogo.vue'

describe('BaseAppLogo.vue', () => {
  it('should match snapshot', () => {
    let wrapper;
    const logo = {
      src: 'https://app.akaunting.com/public/img/akaunting-logo-white.svg',
      description: 'Akaunting-logo'
    }

    wrapper = shallowMount(BaseAppLogo, {
      propsData: {
        logo
      }
    })

    expect(wrapper.element).toMatchSnapshot();
  });
});
