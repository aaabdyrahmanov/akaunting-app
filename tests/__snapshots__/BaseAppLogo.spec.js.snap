// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseAppLogo.vue should match snapshot 1`] = `
<div
  class="header"
>
  <img
    alt="Akaunting-logo"
    class="mb-3"
    src="https://app.akaunting.com/public/img/akaunting-logo-white.svg"
  />
</div>
`;
